<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout" rowstretch="0,1,10">
      <item row="1" column="0">
       <widget class="QComboBox" name="chooseFunc_cmbbox">
        <item>
         <property name="text">
          <string>memcpy</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>memmove</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>memset</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>strcpy</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>strncpy</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>strcat</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>strncat</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>memcmp</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>strcmp</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>strcoll</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>strncmp</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>strxfrm</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>strtok</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>strerror</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>strlen</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Введите первую строку</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="first_lEdit"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Введите вторую строку</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="second_lEdit"/>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="enter_btn">
          <property name="text">
           <string>Ввод</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QListWidget" name="Info_list"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
